src/transformers/tokenization_utils_base.py

```diff
@@ -1001,7 +1011,9 @@ def add_tokens(
         if not isinstance(new_tokens, (list, tuple)):
             new_tokens = [new_tokens]
 
-        return self._add_tokens(new_tokens, special_tokens=special_tokens)
+        nb_added_tokens = self._add_tokens(new_tokens, special_tokens=special_tokens)
+        # update the cached mapping
```

Comment:
Comments always refer to code on the same line or below 

```suggestion
        # update the cached mapping
        nb_added_tokens = self._add_tokens(new_tokens, special_tokens=special_tokens)
```